### PR TITLE
Fix async/timeout issue in unit tests + simple conf fix

### PR DESCRIPTION
**Pull request by ilya-martynov**

### CODE REVIEW

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

File: conf/logback.xml [hunk 1]
```diff
@@ -26,17 +26,17 @@
 	</appender>
 
 	<logger name="play" level="INFO" />
-	<logger name="application" level="INFO" />
-	<logger name="controllers" level="INFO" />
-	<logger name="models" level="INFO" />
+	<logger name="application" level="DEBUG" />
+	<logger name="controllers" level="DEBUG" />
+	<logger name="models" level="DEBUG" />
```
Comment (dmitriyisaev) [inline conf/logback.xml]: Bad idea. We don't want debug mode by default in production. But we can duplicate logback.xml to test/tesources to have debug mode for tests

**Review by dmitriyisaev — CHANGES_REQUESTED** [review]

File: conf/logback.xml (hunk 1 above)
Comment (ilya-martynov) [reply conf/logback.xml]: We configure which level should be in application.conf. logback is root settings.

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-martynov** [comment]
Seems works well. 
@dmitriyisaev @uralian , could you please finish the review ?

**Comment by vananyev** [comment]
@ilya-martynov , does your changes fixed commented tests? https://github.com/IOT-DSA/scala-broker/blob/fefcdc87b982dc15005697f65afe06c57f89aaf0/test/models/akka/local/LocalDSLinkFolderActorSpec.scala#L104

**Review by dmitriyisaev — APPROVED** [review]

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by vananyev** [comment]
@ilya-martynov , all tests passed from my side!

**Review by vananyev — APPROVED** [review]

**Pull request merged**